### PR TITLE
Add color to plan in main Logo

### PR DESCRIPTION
**Pull request by econdepe**
Use new classes from design-system (https://design-system-beta.netlify.com/colors/)

### CODE REVIEW

File: components/logo/MailLogo.js [hunk 1]
```diff
@@ -38,7 +45,7 @@ const MailLogo = ({ planName = '' }) => {
 };
 
 MailLogo.propTypes = {
-    planName: PropTypes.string
+    planName: PropTypes.oneOf(['', 'plus', 'proffesional', 'visionary'])
```
Comment (EpokK) [inline components/logo/MailLogo.js]: Use constants please.

Comment (EpokK) [reply components/logo/MailLogo.js]: ⚠️ you wrote `proffesional`.

File: components/logo/MailLogo.js [hunk 2]
```diff
@@ -29,7 +29,14 @@ const MailLogo = ({ planName = '' }) => {
...
             <title id="logo__title">ProtonMail</title>
             {planName ? (
-                <text textAnchor="end" className="plan uppercase bold" x="147" y="42" id="plan" focusable="false">
+                <text
+                    textAnchor="end"
+                    className={`plan fill-${planName} uppercase bold`}
+                    x="147"
+                    y="42"
+                    id="plan"
+                    focusable="false"
```
Comment (EpokK) [inline components/logo/MailLogo.js]:
```suggestion
                    focusable={false}
```

File: components/logo/MailLogo.js (hunk 1 above)
Comment (econdepe) [reply components/logo/MailLogo.js]: not my best day...

File: components/logo/MailLogo.js [hunk 3]
```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import { PLAN_NAMES } from 'proton-shared/lib/constants';
+
+const planNames = ['', ...Object.keys(PLAN_NAMES)];
```
Comment (EpokK) [inline components/logo/MailLogo.js]: I don't think you need to add `''` since `planName` prop is not required.

Comment (econdepe) [reply components/logo/MailLogo.js]: mmhh, that's a good point!

**Pull request merged**